lib/main.dart

```diff
@@ -1,65 +1,29 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:qlink/theme.dart';
+import 'package:qlink/screens/screens.dart';
 
 void main() {
-  runApp(MyApp());
+  _setSystemColors();
+  runApp(QlinkApp());
 }
 
-class MyApp extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
-    );
-  }
-}
-
-class MyHomePage extends StatefulWidget {
-  MyHomePage({Key? key, this.title}) : super(key: key);
-
-  final String? title;
-
-  @override
-  _MyHomePageState createState() => _MyHomePageState();
+void _setSystemColors() {
+  SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
+    statusBarColor: QlinkTheme.theme.canvasColor,
+    systemNavigationBarColor: QlinkTheme.theme.canvasColor,
+    systemNavigationBarDividerColor: QlinkTheme.theme.canvasColor,
+  ));
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
-
-  void _incrementCounter() {
-    setState(() {
-      _counter++;
-    });
-  }
-
+class QlinkApp extends StatelessWidget {
```

Comment:
I wonder if people tend to leave this in main, or separate out the base widget into its own file.
I usually separate it out on my own projects.